components/global/global_header.tsx

```diff
@@ -41,27 +26,49 @@ const GlobalHeader = () => {
         return null;
     }
 
+    const theme = {
+        ...lightTheme,
+        noStyleReset: true,
+        noFontFaces: true,
+        noDefaultStyle: true,
+        background: {
+            ...lightTheme.background,
+            shape: 'var(--sidebar-teambar-bg)',
+        },
+    };
+
+    // adding the dark theme for now to test how the colors "perform"
     return (
-        <HeaderContainer>
-            <ProductSwitcher/>
-            <AppSpectificContent>
-                {currentProductID !== null &&
-                    <Pluggable
-                        pluggableName={'Product'}
-                        subComponentName={'headerComponent'}
-                        pluggableId={currentProductID}
+        <>
+            <ThemeProvider theme={theme}>
+                <Grid<PShape>
+                    element={Shape}
+                    columnsTemplate={'auto 1fr auto'}
+                    placeItems={{alignItems: 'center'}}
+                    padding={Spacing.trbl({top: 75, right: 200, bottom: 75, left: 100})}
```

Comment:
This appears to add some padding which could interfere when plugins are trying to integrate.